client/packages/programs/src/JsonForms/common/components/Select.tsx

```diff
@@ -100,7 +102,12 @@ const getDisplayOptions = (
         );
         return prev;
       }
-      prev.push({ value: key, label: value ?? key, description, right });
+      prev.push({
```

Comment:
Need to do this since the default json isn't saved in database and therefore won't be translated